playwright-e2e/utils/test-utils.ts

```diff
@@ -27,9 +27,7 @@ export async function downloadConsentPdf(context: BrowserContext, locator: Locat
   // Use axis to fetch pdf directly
   const downloadHref = await locator.getAttribute('href');
   expect(downloadHref).not.toBeNull();
-  expect(downloadHref).toMatch(
-    new RegExp(/https:\/\/storage\.googleapis\.com\/singular-(dev|staging)-assets\/consent_self.pdf/)
-  );
+  expect(downloadHref).toMatch(new RegExp(/https:\/\/storage\.googleapis\.com\/singular-(dev|staging)-assets\/consent_self.pdf/));
```

Comment:
Is this used for a DSM consent pdf download test?

Reply:
It's used in Singular test as this time, downloading Consent pdf during new enrollment, on "Consent Form" page. But we can change this to be more generic.